File: WarehouseManagementWeb.Application/Dto/Output/ResourceShipment/ResourceShipmentItemOutput.cs
namespace WarehouseManagementWeb.Application.Dto.Output.ResourceShipment
{
    /// <summary>
    /// Класс выходной модели входящих ресурсов отгрузки.
    /// </summary>
    public class ResourceShipmentItemOutput
    {
        /// <summary>
        /// Id ресурса отгрузки.
        /// </summary>
        public int ResourceShipmentId { get; set; }

        /// <summary>
        /// Id ресурса.
        /// </summary>
        public int ResourceId { get; set; }

        /// <summary>
        /// Наименование ресурса.
        /// </summary>
        public string? ResourceTitle { get; set; }

        /// <summary>
        /// Id единицы измерения.
        /// </summary>
        public int MeasureUnitId { get; set; }

        /// <summary>
        /// Наименование единицы измерения.
        /// </summary>
        public string? MeasureUnitTitle { get; set; }

        /// <summary>
        /// Количество ресурса.
        /// </summary>
        public int ResourceQuantity { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/warehouse/document-shipment/models/output/resource-shipment-item-output.ts
/**
 * Класс выходной модели входящих ресурсов отгрузки.
 */
export class ResourceShipmentItemOutput {
  /**
   * Id ресурса отгрузки.
   */
  resourceShipmentId: number = 0;

  /**
   * Id ресурса.
   */
  resourceId: number = 0;

  /**
   * Наименование ресурса.
   */
  resourceTitle: string = '';

  /**
   * Id единицы измерения.
   */
  measureUnitId: number = 0;

  /**
   * Наименование единицы измерения.
   */
  measureUnitTitle: string = '';

  /**
   * Количество ресурса.
   */
  resourceQuantity: number = 0;
}
